src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -658,6 +645,12 @@ struct MKLDNNPostEltwiseParam {
   float alpha = 0.f;
   float beta = 1.f;
 };
+void MKLDNNRun(mxnet::FComputeEx fn,
```

Comment:
better to have a blank line before this function :)

Reply:
@rongzha1

Reply:
done